### PR TITLE
Go Daddy domain connection

### PR DESCRIPTION
**Pull request by emranlatif**
Connection to godaddy domains

### CODE REVIEW

File: userdashboard.pixely.com.arecord.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+    "providerId": "userdashboard.pixely.com",
+    "providerName": "Pixely",
+    "serviceId": "arecord",
+    "serviceName": "Managed WordPress Hosting",
+    "version": 2,
+    "logoUrl": "https://www.domainconnect.org/wp-content/uploads/2021/06/flywheel-logo.png",
```
Comment (pawel-kow) [inline userdashboard.pixely.com.arecord.json:7]: Copy-paste issue with logoUrl?

File: userdashboard.pixely.com.arecord.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
+    "providerId": "userdashboard.pixely.com",
+    "providerName": "Pixely",
+    "serviceId": "arecord",
+    "serviceName": "Managed WordPress Hosting",
+    "version": 2,
```
Comment (pawel-kow) [inline userdashboard.pixely.com.arecord.json:6]: Version 1?

**Review by pawel-kow — CHANGES_REQUESTED** [review]

**Comment by emranlatif** [comment]
Hi,

Please add this as a template and let us know. We will than coordinate with
domain providers to make further changes.

{
    "providerId": "userdashboard.pixely.com",
    "providerName": "Pixely",
    "serviceId": "arecord",
    "serviceName": "Managed WordPress Hosting",
    "version": 2,
    "logoUrl": "https://userdashboard.pixely.com/wp-content/plugins/admin-2020/admin/assets/img/LOGO-BLUE.png",
    "description": "Configure ip address for the domain A record.",
    "syncBlock": false,
    "syncPubKeyDomain": "domainconnect.userdashboard.pixely.com",
    "syncRedirectDomain": "userdashboard.pixely.com",
    "records": [
        {
            "type": "A",
            "host": "@",
            "pointsTo": "%ip%",
            "ttl": 3600
        }
    ]
}


Regards,
Imran

**Comment by pawel-kow** [comment]
@emranlatif Please update the pull request with the corrected template, or close this one and open a new.

**Comment by emranlatif** [comment]
Hi,

Its not allowing me to push the changes from my git. And because of that I
cannot create a PR.

my git email is ***@***.***

Regards,
Imran

**Comment by pawel-kow** [comment]
@emranlatif not sure how I can help on this. I guess you need to get access back to your github and update this PR or create a new one.

**Comment by emranlatif** [comment]
I just created a new PR. Please acknowledge.

Regards,
Imran

**Comment by pawel-kow** [comment]
You should push to your own fork, not to the upstream project

**Pull request closed without merging**